src/library/autodj/autodjfeature.h

```diff
@@ -92,6 +93,7 @@ class AutoDJFeature : public LibraryFeature {
     QSignalMapper m_crateMapper;
 
     QIcon m_icon;
+    WLibrarySidebar *m_pSidebarWidget;
```

Comment:
Nit pick: * one position left.

Can we gurantee that the lifetime of this class is longer than the sidebar? I think yes.
Can you add a comment explaining this?

Reply:
you mean, explain why it's not `AutoDJFeature::~AutoDJFeature()`?

Reply:
I had something in mind like 

// This is a non owning pointer that outlives this object.

But I just got aware that thid his is not true if you switch the skins. 
I think it is most appropriated to use a QPointer here that is exactly for this purpose. 
Than there is no need for an extra comment.

Reply:
I think this comment still applies. Can you use here: 
QPointer<WLibrarySidebar> m_pSidebarWidget;
You can than remove also the nullptr initalizer, because it is automatically initialized.

Reply:
ah ok, now I got it.